documentation/airport-guides/kaus.md

```diff
@@ -1,16 +1,20 @@
 ### About this facility
+
 Austin ATCT (AUS), Austin, TX, USA  
 199,632 aircraft movements in 2017 ([FAA ATADS](https://aspm.faa.gov/opsnet/sys/Airport.asp))
 
 This is the Control Tower position at AUS, serving Austin-Bergstrom International Airport, the main airport for Austin, TX.
 
 ### Airspace
+
 Class C airspace extends to 17,000 feet above sea level, encompassing the area around the airport.
 
 ### Instrument Procedures
+
 Traffic to/from the West is usually assigned to RWY17R/35L Traffic to/from the East is usually assigned to RWY17L/35R. During the hours of 0600 and 1200, landing on the 35's, and takeoffs on the 17's are preferred. For cargo aircraft, 17R/35L is preferred for both takeoffs and landings.

```

Comment:
By this I meant on this line.
![image](https://user-images.githubusercontent.com/19408472/42184526-c2611a3a-7e3d-11e8-98c8-ddd2e8d77fb7.png)

As you can see from the image, that's incredibly ugly to read; you should lay it out as it was in my original image.